pkg/model/dev.go

```diff
@@ -91,6 +91,7 @@ type Dev struct {
 	Namespace            string                `json:"namespace,omitempty" yaml:"namespace,omitempty"`
 	Container            string                `json:"container,omitempty" yaml:"container,omitempty"`
 	Image                string                `json:"image,omitempty" yaml:"image,omitempty"`
+	Push                 *BuildInfo            `json:"-,omitempty" yaml:"push,omitempty"`
```

Comment:
Yes!!!!

should we call it `build` since it's buildinfo?

Reply:
and could we also use it in `build`?

Reply:
I talked to @rlamana  and the idea is to have a `build` field for building the dev image. `okteto up --build` would build your dev image on the fly. This is why we chose `push` here.

Reply:
cool